datacube/index/abstract.py

```diff
@@ -1921,6 +1923,88 @@ def init_db(self,
         :return: true if the database was created, false if already exists
         """
 
+    # Spatial Index API
+
+    def create_spatial_index(self, crs: CRS) -> bool:
+        """
+        Create a spatial index for a CRS.
+
+        Note that a newly created spatial index is empty.  If there are already datatsets in the index whose
+        extents can be safely projected into the CRS, then it is necessary to also call update_spatial_index
+        otherwise they will not be found by queries against that CRS.
+
+        Only implemented by index drivers with supports_spatial_indexes set to True.
+
+        :param crs: The coordinate reference system to create a spatial index for.
+        :return: True if the spatial index was successfully created (or already exists)
+        """
+        if not self.supports_spatial_indexes:
+            raise NotImplementedError("This index driver does not support the Spatial Index API")
+        else:
+            raise NotImplementedError()
+
+    def spatial_indexes(self, refresh=False) -> Iterable[CRS]:
```

Comment:
Same comment re: naming as `spatial_indexes` in _spatial.py

Reply:
Oh OK, I've reverted, and renamed `spatial_indexes` to `spatially_indexed_crses`